crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -235,6 +296,14 @@ pub mod error {
         }
     }
 
+    #[derive(Debug, thiserror::Error)]
+    pub enum Solution {
```

Comment:
Would be nice if the errors contained the order or price that could not be found.

Reply:
added